docs/CODE_STYLE.md

```
@@ -1,60 +1,85 @@
-- **Functions** don’t need prefixes but should:
-  - Use at least two words to avoid overlap with instance variables (e.g., `draw_something()`).
-  - Name functions as actions where possible (`draw_something()` vs. `green_apple()`).
+**Global Variables**:
+- Require no additional prefix, as they already use `global.`.
+- Example: `global.example_e1`.
 
-- **Macro** constants:
-  - To denote their group, use a short prefix in all caps (e.g., `PREFIX_`).
-  - After the prefix, use `snake_case`.
-  - Examples: `COL_bright_red`, `DIR_left`.
+### Functions
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)